superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -68,7 +68,7 @@ def _process_numeric_result(self, metric, result):
             return False
 
         self._result.add_raw_data(metric, result)
-        self._result.add_result(metric, sum(result) / len(result))
+        self._result.add_result(metric, sum(result) / len(result), reduce_type)
```

Comment:
built-in `sum` has [approximation issue](https://docs.python.org/3/tutorial/floatingpoint.html), e.g.,
```
>>> sum([0.1, 0.1, 0.1])
0.30000000000000004
```
```suggestion
        self._result.add_result(metric, statistics.mean(result), reduce_type)
```

Reply:
I will correct it in a separate PR. Thanks!